meson.build

```diff
@@ -22,15 +23,19 @@ import('pkgconfig').generate(
 	name: meson.project_name(),
 	description: 'JIT assembler for x86(IA32), x64(AMD64, x86-64)',
 	version: meson.project_version(),
-	url: 'https://github.com/herumi/xbyak'
+	url: 'https://github.com/herumi/xbyak',
+	variables: ['includedir=@0@'.format(include_dir)],
+	dataonly: true,
 )
 
+shared_cmake_dir = get_option('prefix') / 'share/cmake/xbyak'
 if meson.version().version_compare('>=0.50.0')
 	cmake = import('cmake')
 
 	cmake.write_basic_package_version_file(
 		name: meson.project_name(),
-		version: meson.project_version()
+		version: meson.project_version(),
+		install_dir: shared_cmake_dir,
 	)
```

Comment:
Installing the version file to `$datadir/cmake` isn't enough to make it arch independent - you also have to specify [`arch_independent: true`](https://mesonbuild.com/CMake-module.html#cmakewrite_basic_package_version_file), otherwise a file built on an amd64 machine won't be usable on e.g. i386. In doing do, the `if meson.version().version_compare('>=0.50.0')` should be bumped to `'>=0.62.0'`